package.json

```diff
@@ -8,13 +8,26 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "~38.0.8",
+    "@react-native-community/masked-view": "^0.1.10",
+    "expo": "^38.0.8",
+    "expo-asset": "~8.1.7",
     "expo-status-bar": "^1.0.2",
     "react": "~16.11.0",
     "react-dom": "~16.11.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz",
+    "react-native-animatable": "^1.3.3",
     "react-native-elements": "^2.0.4",
-    "react-native-web": "~0.11.7"
+    "react-native-gesture-handler": "^1.7.0",
+    "react-native-keyboard": "^2.0.1",
+    "react-native-keyboard-aware-scroll-view": "^0.9.1",
+    "react-native-reanimated": "^1.10.1",
+    "react-native-safe-area-context": "^3.1.1",
+    "react-native-screens": "^2.9.0",
```

Comment:
Are you still using `react-native-screens` and `react-native-animated` in your code? If not, remove these packages with npm or yarn.

Reply:
Both the packages are actually dependencies for `react-navigation-stack`.